### ISSUE TITLE
transfer information from SLL GDPR for Life Science data page

### ISSUE THREAD

**Issue by elinkronander** [ScilifelabDataCentre/RDM-guidelines#172]
The aim is to transfer the guidelines found on SLL GDPR for Life Science data https://www.scilifelab.se/data/gdpr-for-life-science-data. 

- [x] Check what is already covered on [gdpr-legal-reference.md](https://github.com/ScilifelabDataCentre/RDM-guidelines/blob/main/content/topics/gdpr-legal-reference.md)
- [x] Make sure remaining relevant guidelines ends up on appropriate pages (see for example #166 )

**Comment by annaasklof** [comment]
The parts "For any set of personal data you are processing think about these questions:" and "For anyone processing personal data, this means:" on the page: https://www.scilifelab.se/data/gdpr-for-life-science-data are not quite covered by: https://data-guidelines.scilifelab.se/topics/gdpr-ethical-review-glossary/#gdpr-general-data-protection-regulation. These are good and we might want to add them on RDM the question is where.

**Comment by YvonneKallberg** [comment]
I made a suggestion in a branch, where I added mainly the 'this means' as bullets under the questions already listed under GDPR considerations. Check that I've captured all of importance. I'm uncertain if the consent part should be there though, since consent is not the legal basis and hence has nothing to do with GDPR but rather ethical consideration. It should probably be moved (since there is a section on Ethical considerations).
